### PR TITLE
Make Jumps work for manual instumentaion

### PR DESCRIPTION
**Pull request by akopich**
Test: Manual
Bug: http://b/228430650

### CODE REVIEW

**Review by karupayun — APPROVED** [review]
LGTM! :)

**Review by dpallotti — APPROVED** [review]

**Review by beckerhe — APPROVED** [review]
LGTM

File: src/OrbitGl/TimeGraph.h [hunk 1]
```diff
@@ -99,19 +99,19 @@ class TimeGraph final : public orbit_gl::CaptureViewElement,
...
                            JumpScope jump_scope);
-  [[nodiscard]] const orbit_client_protos::TimerInfo* FindPreviousFunctionCall(
-      uint64_t function_address, uint64_t current_time,
+  [[nodiscard]] const orbit_client_protos::TimerInfo* FindPreviousScopeCall(
+      uint64_t scope_id, uint64_t current_time,
       std::optional<uint32_t> thread_id = std::nullopt) const;
-  [[nodiscard]] const orbit_client_protos::TimerInfo* FindNextFunctionCall(
-      uint64_t function_address, uint64_t current_time,
+  [[nodiscard]] const orbit_client_protos::TimerInfo* FindNextScopeCall(
+      uint64_t scope_id, uint64_t current_time,
```
[the remark this answers is not in the document]
Comment (akopich) [reply src/OrbitGl/TimeGraph.h:106]: Maybe FindNextScopeTimer? As FindNextScope suggests, a timer corresponding to a different scope will necessarily be returned.

[the remark this answers is not in the document]
Comment (dpallotti) [reply src/OrbitGl/TimeGraph.h:106]: Sounds good!

**Pull request merged**